systems/daint-gpu.sh

```diff
@@ -40,9 +40,10 @@ ns_makej=20
 ns_threads_per_core=2
 ns_cores_per_socket=12
 ns_sockets=1
-ns_threads_per_socket=24
+ns_threads_per_socket=12
 
 run_with_mpi() {
-    echo ARB_NUM_THREADS=$ns_threads_per_socket srun -n $ns_sockets -N $ns_sockets -c $ns_threads_per_socket "${@}"
-    ARB_NUM_THREADS=$ns_threads_per_socket srun -n $ns_sockets -N $ns_sockets -c $ns_threads_per_socket "${@}"
+    export ARB_NUM_THREADS=$ns_threads_per_socket
+    echo srun -n1 -N1 -c $ns_threads_per_socket "${@}"
+    srun -n1 -N1 -c $ns_threads_per_socket "${@}"
 }
```

Comment:
Should have a comment here explaining why these numbers are hard coded, or to use the `ns_sockets` variable etc.